src/WeatherWidget/WeatherWidget.js

```diff
@@ -0,0 +1,61 @@
+import './WeatherWidget.scss';
+import cloudy from '../assets/cloudy.svg';
+
+function WeatherWidget() {
```

Comment:
This widget looks awesome! I know a lot of this will have to be updated when we get the actual API, but it looks great.